src/components/NavBar/MenuItem/index.tsx

```diff
@@ -42,6 +47,20 @@ function MenuItem({ classes, title, icon, layersCategories }: MenuItemProps) {
     dispatch(loadTable(table.id));
   };
 
+  const toggleLayers = (layers: LayerType[]) => {
```

Comment:
Can you explain a bit more what's happening here? Is this function specific to groupedLayers?